urmila/HTML/Assignment7/sign.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <style>
+        body {
+            background-color: aqua;
+            background-repeat: no-repeat;
+            background-size: cover;
+            background-size: 1000px 1000px;
+            background-attachment: fixed;
+        }
+    </style>
+    <div>
+        <div>
+            <div>
+                <h1>Find your job with us</h1>
+            </div>
+            <div>
+                <div>
+                    <ul>
+                        <a href="http://homepage.html" target="_blank"><button>Home</button></a>
+                        <a href="http://login.html" target="_blank"><button>Login</button></a>
+                        <a href="#0.1_contact"><button>Sign up</button></a>
+                    </ul>
+                </div>
+            </div>
+        </div>
+        <div>
+            <p>Please enter your Email and password</p>
+            <form action="http://profile.html" target="_blank"
+                onsubmit="try {return window.confirm(&quot;This form may not function properly due to certain security constraints.\nContinue?&quot;);} catch (e) {return false;}">
+                <label>Email:</label>
+                <input type="email" name="email"><br><br>
+                <label>Password:</label>
+                <input type="password" name="pwd"><br><br>
+                <p>Please enter the password once again for conformation</p>
+                <label>Password:</label>
+                <input type="password" name="cnpwd"><br><br><br><br>
+                    <button type="submit"> Signup</button>
+            </form>
+        </div>
+    </div>
+    </div>
+    </body>
+</html>
```

Comment:
make proper indentation 
make sure it before pushing and check that all diva have closing tag for them